wolf/cmake/stream.cmake

```diff
@@ -125,25 +125,34 @@ if (WOLF_STREAM_RIST)
     
     set(FETCHCONTENT_QUIET OFF)
     FetchContent_MakeAvailable(${RIST_TARGET})
+
+    # this file should be available after building rist
+    set(RIST_LIBRARY_BINARY_FILE ${rist_BINARY_DIR}/librist.a)
       
     if (ANDROID)
-      add_custom_command(OUTPUT rist_command.out COMMAND
-      /bin/bash "${CMAKE_CURRENT_SOURCE_DIR}/third_party/shells/librist/librist-android.sh" --build_dir=${rist_BINARY_DIR}
-       WORKING_DIRECTORY ${rist_SOURCE_DIR})
-      add_custom_target(rist ALL DEPENDS rist_command.out)
-      
-      list(APPEND LIBS
-        ${rist_BINARY_DIR}/librist.a)
-    else ()
-      STRING(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_LOWER)
-      
-      add_custom_command(OUTPUT rist_command.out COMMAND cmd /c "meson setup ${rist_BINARY_DIR} --backend vs2022 --default-library static --buildtype ${CMAKE_BUILD_TYPE_LOWER} & meson compile -C ${rist_BINARY_DIR}" WORKING_DIRECTORY ${rist_SOURCE_DIR})
-      add_custom_target(rist ALL DEPENDS rist_command.out)
+        add_custom_command(OUTPUT ${RIST_LIBRARY_BINARY_FILE}
```

Comment:
As mentioned above, please use the middle name 'rist_command.out' as output and use it in the 'add_custom_target'.

Reply:
okay. I'm not sure what you mean by "middle name", so I'll add it along the library file as output.

Reply:
Leave your code unchanged. Please refer to the above comment.

Reply:
sure, as you say so.